IrisSoftware/UI.py

```diff
@@ -1,23 +1,18 @@
 """Handles running the UI elements."""
 import sys
 from PySide6.QtWidgets import QApplication
-from . import widgets
+from IrisSoftware.widgets import MainWidget
 
 CALIBRATION_FILE_NAME = "calibrationData.pickle"
 
 
 class UI:
```

Comment:
It looks like the UI class doesn't do very much other than calling `mainWidget.show()` and `app.exec()`. Could we simplify our file structure by moving the connect callbacks into the MainWidget class and moving the QApplication to main.py? We could even rename MainWidget to UIWidget or UI or something so it's clear what it does.

Reply:
This is similar to the Camera class, but my idea for this was that the `IrisSoftware` class would only interact with the `UI` class. I do anticipate expanding the `UI` class to include opening the `CalibrationWidget` and potentially the settings page, but that will be another PR. If we wanted to simplify our file structure a bit, I could move the widgets into the `UI.py` file if that works, since they are directly related to the UI. What do you think?

Reply:
If you have plans to add more features to `UI` so that the `UI` class serves a separate logical role than the `MainWidget` class, then it probabally makes sense to keep them separate. But it might be good to rename `UI.py` to `ui.py` since the rest of our files start with lowercase.